code/API_definitions/NetworkSliceBooking.yaml

```
@@ -0,0 +1,430 @@
+openapi: 3.0.3
+info:
+  title: API for Network Slice Booking
+  description: |
+    The Network Slice Booking (NSB) API provides programmable interface for developers to reserve a slice resource of a selected area within a period, and manage device access control as needed.
+  version: wip
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+servers:
+  - url: "{apiRoot}/nsb/vwip"
+    variables:
+      apiRoot:
+        default: http://localhost:9100
+tags:
+  - name: NSB Sessions

```

Please avoid abbreviation ... especially here in the tags it makes the API spec less readable (the tags are only for readability there)
```suggestion
  - name: Network Slice Booking Sessions
```